css/style.css

```diff
@@ -61,12 +69,71 @@ input {
   text-align: center;
 }
 
-ul li:nth-child(odd) {
+#booksDisplay ul li:nth-child(odd) {
   background-color: rgb(175, 175, 175);
   color: white;
 }
 
-ul li:nth-child(even) {
+#booksDisplay ul li:nth-child(even) {
   background-color: white;
   color: black;
 }
+
+nav {
+  padding: 0;
+  margin: 0;
+}
+
+.navbar {
+  display: flex;
+  justify-content: flex-end;
+  list-style: none;
+  background-color: rgb(208, 204, 204);
+  padding: 0;
+  margin: 0;
+}
+
+.navbar a {
+  text-decoration: none;
+  color: black;
+  font-weight: bold;
+}
+
+.hideSection {
+  display: none;
+}
+
+#contact {
+  position: relative;
+  text-align: center;
+}
+
+#contact ul {
+  text-align: justify;
+  margin: auto;
+  margin-top: 20px;
+  display: flex;
```

Comment:
- [ ] [OPTIONAL] Let's strive to keep the code DRY. This property:
```CSS
display: flex
```
has been used several times in the CSS. Please refactor it to a class and apply the class in the HTML file to elements that share the same display property.

### Example code
```CSS
.d-flex {
  display: flex;
  align-items: center;
}
```

Then apply this class to HTML element that needs this behavior:

```HTML
  <section class="d-flex">
```